### PR TITLE
added reviews, slightly alterning CSS as I go, lots to change still

### PR DESCRIPTION
**Pull request by chriscodingxyz**
added tailwind, no major css updates yet, just got reviews going for now

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,33 @@
...
+import Footer from "./Components/Footer";
+import Header from "./Components/Header";
 
 function App() {
-  return <div className="App">test</div>;
+  const [isLoading, setIsLoading] = useState(true);
+  const [reviews, setReviews] = useState([]);
+
+  useEffect(() => {
+    // setIsLoading(true);
```
Comment (roisinoreilly) [inline src/App.js:13]: did you mean to comment these out?

File: src/App.js [hunk 2]
```diff
@@ -1,7 +1,33 @@
...
+    getReviews().then((data) => {
+      console.log(data, "dat");
+      setReviews(data);
+      // setIsLoading(false);
+    });
+  }, []);
+
+  console.log(reviews, "reviews here");
```
Comment (roisinoreilly) [inline src/App.js:21]: don't forget to remove console.logs

File: src/Components/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export default function Footer() {
+  return (
+    <div className="Footer fixed bottom-0 left-0 right-0 bg-gray-300 vh10 Footer">
+      FOOTER / Add scroll to top feature
```
Comment (roisinoreilly) [inline src/Components/Footer.jsx:4]: excited to see this!

File: src/Components/ReviewCard.jsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+      <img src={review_img_url} alt={title} />
+      <ul>
+        <li>Owner: {owner}</li>
+        <li>Title: {title}</li>
+        <li>Review ID: {review_id}</li>
+        <li>Category: {category}</li>
+
+        <li>votes: {votes}</li>
+        <li>designer: {designer}</li>
+        <li>comment count: {comment_count}</li>
```
Comment (roisinoreilly) [inline src/Components/ReviewCard.jsx:23]: I think just comments as opposed to comment count is more user friendly and what a user would expect to see

File: src/Components/ReviewCard.jsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  designer,
+  comment_count,
+}) {
+  return (
+    <div className="ReviewCard border border-black border-2 max-w-xs">
+      <img src={review_img_url} alt={title} />
+      <ul>
+        <li>Owner: {owner}</li>
+        <li>Title: {title}</li>
+        <li>Review ID: {review_id}</li>
```
Comment (roisinoreilly) [inline src/Components/ReviewCard.jsx:18]: I don't think you need to display the ID to the user

File: src/Components/Reviews.jsx [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import ReviewCard from "./ReviewCard";
+
+export default function Reviews({ reviews }) {
+  return (
+    <div className="Reviews flex flex-wrap gap-4">
+      {reviews.map((review) => {
+        return <ReviewCard key={crypto.randomUUID()} {...review} />;
```
Comment (roisinoreilly) [inline src/Components/Reviews.jsx:8]: interesting! Is there any data we already have access to in `review` that's unique to each review? Remember keys don't need to be unique across the entire app so in this case a review id would be fine

File: src/api.js [hunk 7]
```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+
+const baseAPI = axios.create({
+  baseURL: `https://nc-games-lh3e.onrender.com/api`,
+});
+
+export const getReviews = () => {
+  return baseAPI.get("/reviews").then((res) => {
+    console.log(res, "this is res api.js");
```
Comment (roisinoreilly) [inline src/api.js:9]: take this one out too

**Review by roisinoreilly — APPROVED** [review]
looks really great! I've left a couple of small comments but PR approved :) merge when ready

File: src/Components/Reviews.jsx (hunk 6 above)
Comment (chriscodingxyz) [reply src/Components/Reviews.jsx:8]: cool, updated to review.review_id

**Pull request merged**